### DISCUSSION TITLE
How to deta micro initilialized

### DISCUSSION THREAD

**Discussion by wei886-tw** [deta/cloud-docs#649] [Help]
I followed the instruction step by step, when I'm doning deploy local changes, shows Error: no deta micro initilialized, see `deta new --help` to create a micro. I'm a Mac user, please teach me how to initialize deta micro

**Comment by abdelhai** [comment]
run `deta new`

**Reply by wei886-tw** [reply]
Terminal shows"No entrypoint file found in '/Users/sam'. Please, provide a name or path to create a new micro elsewhere. See `deta new --help`.'"

**Comment by AyushSehrawat** [comment]
https://basic.deta.dev will be of help to you
